src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/EmailAddressAdder.tsx

```diff
@@ -94,6 +97,16 @@ const EmailAddressAddForm = () => {
     }
   }, [formState, recordTelemetry]);
 
+  const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setEmail(e.target.value);
+  };
+
+  const isEmailValid = () => {
+    // Regex for checking email format
```

Comment:
:exclamation:  Regex alert :exclamation: 

Regexes are notoriously hard to read, so to save future us (and present-day PR reviewers :sweat_smile:) some headaches, it can be helpful to add a comment describing what it actually does ([example](https://github.com/mozilla/blurts-server/blob/1f6bc47d79df559189afe93d99329d0187700aa7/src/app/(proper_react)/(redesign)/(public)/LandingView.test.tsx#L300-L309)).

After looking at the docs, I think this checks for more than 1 non-whitespace-non-@ characters, followed by an `@`, and then again more than one of those characters, followed by a `.`, and one more group of those characters? e.g. `x@x.x` passes, and `x@x`, `x.x`, `@x.x` don't?